x-pack/plugins/maps/public/classes/styles/vector/components/legend/vector_style_legend.js

```diff
@@ -4,18 +4,25 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { Fragment } from 'react';
+import React from 'react';
 
 export function VectorStyleLegend({ isLinesOnly, isPointsOnly, styles, symbolId }) {
-  return styles.map((style) => {
-    return (
-      <Fragment key={style.getStyleName()}>
-        {style.renderLegendDetailRow({
-          isLinesOnly,
-          isPointsOnly,
-          symbolId,
-        })}
-      </Fragment>
-    );
-  });
+  const legendRows = [];
+  for (let i = 0; i < styles.length; i++) {
```

Comment:
Getting an un-keyed element in the array.

<img width="300" alt="Screen Shot 2020-05-26 at 8 18 04 AM" src="https://user-images.githubusercontent.com/373691/82911788-8ca01b80-9f29-11ea-9688-08583ef6d362.png">